docs/sources/setup/configuration.md

```diff
@@ -58,3 +58,13 @@ If you want your data source to connect via proxy, set the environment appropria
 If you want to setup specific proxy URL for the datasource, you can configure in the datasource config network section.
 
 > Proxy URL specification in data source config is available from v2.2.0
+
+## Allowing non HTTP GET / POST methods
+
+By default infinity only allow GET/POST HTTP methods to reduce the risk of destructive/accidental payloads. But through configuration, you can allow other methods (`PATCH`,`POST` and `DELETE`) for any unconventional use cases. If you need to make use of this feature, Enable the `Allow non GET / POST HTTP verbs` setting under URL section of the datasource config
```

Comment:
```suggestion
By default infinity only allow GET/POST HTTP methods to reduce the risk of destructive/accidental payloads. But through configuration, you can allow other methods (`PATCH`,`POST` and `DELETE`) for any unconventional use cases. If you need to make use of this feature, Enable the `Allow non GET / POST HTTP methods` setting under URL section of the datasource config
```

I would stick to `methods` as more used, and not switch between `methods` and `verbs`.